specs/commandline/policy.md

```diff
@@ -0,0 +1,671 @@
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  add           Add trust policies.
+  delete        Delete trust policies.
+  list          List trust policies.
+  update        Update trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy add
+
+```text
+Add trust policies.
+
+Usage:
+  notation policy add [flags] <policy_name>
+
+Flags:
+      --cert-file          string   optional certificate filepath
+      --custom-level       string   optional custom level to based existing verification level, e.g. "authenticity=log,expiry=log"
+  -h, --help                        help for add
+      --id                 string   optional trust identity aka certificate subject info
+      --repo               string   optional repository URI (default "*")
+      --trust-store        string   required trust store in format "<trust_store_type>:<trust_store_name>", e.g. "ca:my_trust_store"
+      --verification-level string   optional verification level, options: "strict", "permissive", "audit", "skip" (default "strict")
+```
+
+### notation policy delete
+
+```text
+Delete trust policies. User cannot delete all the trust policies, at least one trust policy should be configured for signature verification.
+
+Usage:
+  notation policy delete [flags] <policy_name>...
+
+Flags:
+  -h, --help   help for delete
+```
+
+### notation policy list
+
+```text
+List trust policies by names
+
+Usage:
+  notation policy list [flags]
+
+Aliases:
+  list, ls
+
+Flags:
+      --cert-file   string       optional certificate filepath
+      --details                  optional list the details of trust policies
+  -h, --help                     help for list
+      --id          string       optional trust identity aka certificate subject info
+      --name        string       optional trust policy name
+      --reference   string       optional reference to the artifact
+      --repo        string       optional repository URI, e.g. localhost:5000/namespace/repo_name
+      --trust-store string       optional trust store in format "<trust_store_type>:<trust_store_name>", e.g. "ca:my_trust_store"
+```
+
+### notation policy update
+
+```text
+Update the existing trust policies.
+
+Usage:
+  notation policy update [flags] <policy_name>
+
+Flags:
+      --cert-file          string   optional certificate filepath
+      --custom-level       string   optional custom level to based existing verification level, e.g. "authenticity=log,expiry=log"
+  -h, --help                        help for add
+      --id                 string   optional trust identity aka certificate subject info
+      --repo               string   optional repository URI
+      --trust-store        string   optional trust store in format "<trust_store_type>:<trust_store_name>", e.g. "ca:my_trust_store"
+      --verification-level string   optional verification level, options: strict, permissive, audit, skip
```

Comment:
```suggestion
     --custom-level       stringArray   {key}={value} pairs that represent a custom level to override existing verification level
-h,  --help                             help for add
     --id                 stringArray   trust identities, user trusted signing certificate subjects
     --scope              stringArray   repository URIs to which the policy applies
     --trust-store        stringArray   trust stores in format <trust_store_type>:<trust_store_name>
     --verification-level string        verification level, options: "strict", "permissive", "audit", "skip"
```
IMO, `notation policy` command's logic should be separated from `notation cert`. The `--cert-file` flag in `notation policy` command might confuse users between these two commands.